src/Task2/MyList.java

```diff
@@ -0,0 +1,40 @@
+package Task2;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class MyList<T extends Comparable> {
```

Comment:
Please, don't use raw types in your code. Set 'T' as a parameter of the Comparable interface `Comparable<T>`